Index: blowfish.h
===================================================================
--- blowfish.h	(revision 306272)
+++ blowfish.h	(working copy)
@@ -40,7 +40,7 @@
 #define _BLF_H_

 /* Schneier states the maximum key length to be 56 bytes.
- * The way how the subkeys are initalized by the key up
+ * The way the subkeys are initalized by the key up
  * to (N+2)*4 i.e. 72 bytes are utilized.
  * Warning: For normal blowfish encryption only 56 bytes
  * of the key affect all cipherbits.
